floodwatch/src/js/components/GraphParent.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Row, Col } from 'react-bootstrap';
+import * as d3 from 'd3';
+import _ from 'lodash'
+import {Chart} from './Chart'
+
+type PropsType = {
+  data: [],
```

Comment:
This is `Array<Array<StackedData>>` - you'll need to import the type `StackedData`.